awsbatch-cli/setup.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from setuptools import find_packages, setup
+
+
+def readme():
+    """Read the README file and use it as long description."""
+    with open(os.path.join(os.path.dirname(__file__), "README")) as f:
+        return f.read()
+
+
+VERSION = "1.0.0"
+REQUIRES = [
+    "setuptools",
+    "boto3>=1.16.14",
+    "tabulate>=0.8.2,<0.8.10",
```

Comment:
tabulate 0.8.10 is not on Pypi. Shall we pin the major directly with something like `tabulate~=0.8`?

Reply:
Done